Elsewhen/Views/MykeMode.swift

```diff
@@ -317,8 +371,19 @@ struct MykeMode: View, OrientationObserving {
             NavigationView {
                 #if os(iOS)
                 TimezoneChoiceView(selectedTimeZone: .constant(TimeZone.current), selectedTimeZones: $selectedTimeZones, selectedDate: $selectedDate, selectMultiple: true)
-                    .navigationBarItems(trailing:
-                                            Button(action: {
+                    .navigationBarItems(leading: Button(action: {
+                        self.alertMessage(title: "Save as a group?", message: "Provide a name for the Time Zone group below.") { action, text in
```

Comment:
Save as a Group with capital G? Although sometimes it's list sometimes group. What do we prefer?